circle.yml

```diff
@@ -1,6 +1,27 @@
 machine:
   node:
     version: 6.2.1
+  environment:
```

Comment:
Is there any reason to do a manual install of yarn for circle? If yarn is included as a dev dependency you should be able to run it from circle I believe.